src/config/config.ts

```diff
@@ -7,10 +7,12 @@ export const NETWORK = {
   name: 'iExec Sidechain',
   url: 'https://bellecour.iex.ec',
 };
-export const DEFAULT_CONTRACT_ADDRESS =
+export const DEFAULT_DATAPROTECTOR_CONTRACT_ADDRESS =
   '0x3a4Ab33F3D605e75b6D00A32A0Fa55C3628F6A59';
 export const DEFAULT_SUBGRAPH_URL =
   'https://thegraph-product.iex.ec/subgraphs/name/bellecour/dataprotector';
 export const WORKERPOOL_ADDRESS = 'prod-v8-bellecour.main.pools.iexec.eth';
 export const SCONE_TAG = ['tee', 'scone'];
 export const DEFAULT_MAX_PRICE = 0;
+export const WHITELIST_SMART_CONTRACT_ADDRESS =
+  '0x781482C39CcE25546583EaC4957Fb7Bf04C277D2';
```

Comment:
this is the web3mail whitelist address, dataprotector should not depend on web3mail